pitch-io/cljest
# Enable Jest to understand Clojure dependencies

This PR adds a dependency extractor and a haste implementation to allow Jest to understand the dependencies a Clojure(Script) file has. 

# Testing

- For the project you're going to run the Jest tests against, update `jest-preset-shadow` in its `package.json`, changing the SHA to the latest commit SHA for this PR.
- Run `yarn`
- Start the `cljest` server separately
- Run `yarn jest:watch`
- Initially, no files should be detected.
- Make a change to some file that's a indirect dependency of a test (a dependency of a dependency).
- It should be picked up and Jest should start running tests
